lib/langchain/output_parsers/structured.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require "json"
+require "json-schema"
+
+module Langchain::OutputParsers
+  # = Structured Output Parser
+  #
+  class StructuredOutputParser < Base
+    attr_reader :schema
+
+    #
+    # Initializes a new instance of the class.
+    #
+    # @param schema [JSON::Schema] The json schema
+    #
+    def initialize(schema:)
+      @schema = validate_schema(schema)
+    end
+
+    def to_h
+      {
+        _type: "StructuredOutputParser",
+        schema: @schema.to_json
```

Comment:
Since you've got the `attr_reader :schema` at the top, this can just be `schema: schema.to_json`

Reply:
Thanks for feedback, i'll fix it.